src/Sylius/Bundle/CoreBundle/Migrations/Version20220127150747.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220127150747 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'A table field instead of using the prices from ChannelPricing while processing the cart';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_order_item ADD original_unit_price INT NOT NULL');
```

Comment:
If this column is not null, we need to migrate current data to it. Otherwise, someone's app will collapse

Reply:
I corrected both the migration and `OrderItem.orm.xml` setup